shrine/lib/backdrop.dart

```diff
@@ -193,27 +198,88 @@ class _BackdropState extends State<Backdrop>
   }
 
   void _toggleBackdropLayerVisibility() {
-    _controller.fling(
-        velocity: _frontLayerVisible ? -_kFlingVelocity : _kFlingVelocity);
+    // Call setState here to update layerAnimation update if that's necessary
```

Comment:
Not sure what "update layerAnimation update" means here.

Reply:
Me neither.

Reply:
Done.

Reply:
Typo, sorry!